src/cityreader/cityreader.py

```diff
@@ -16,7 +21,21 @@
 # should not be loaded into a City object.
 cities = []
 
+import csv
 def cityreader(cities=[]):
+    with open('cities.csv', newline='') as csvfile:
+        spamreader = csv.reader(csvfile, delimiter=' ', quotechar='|')
+        cnt = 0
+        for row in spamreader:
+            x = row[0].split(',')
+            if cnt > 0:
+                try:  
+                    cities.append((x[0],int(x[3]),int(x[4])))
+                except:
+                    cnt = cnt      
+            cnt +=1   
+
```

Comment:
Awesome handling here. Had there been a failing form case, your code would have handled it. Like we spoke about, you could have used `next()` here as well.